pyrfa/meta.yaml

```
@@ -0,0 +1,56 @@
+{% set name = "pyrfa" %}
+{% set version = "8.5.3" %}
+{% set fn_base = "{}-{}-cp{}-none".format(name, version, CONDA_PY) %}
+{% set fn_win = "{}-win_amd64.whl".format(fn_base) %}
+{% set fn_linux = "{}-manylinux1_x86_64.whl".format(fn_base) %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ fn_win }}  # [win64]
+  fn: {{ fn_linux }}  # [linux64]
```

This can be used to rename the downloaded file:
```suggestion
  fn: pyrfa.whl
```
(else it isn't needed at all)